src/lib.rs

```diff
@@ -1185,13 +1185,20 @@ where
 /// Create a new builder with the default environment variables.
 ///
 /// The builder can be configured before being initialized.
+/// This is a convenient way of calling [`Builder::from_default_env`].
+///
+/// [`Builder::from_default_env`]: struct.Builder.html#method.from_default_env
 pub fn builder() -> Builder {
     Builder::from_default_env()
 }
 
 /// Create a builder from the given environment variables.
 ///
 /// The builder can be configured before being initialized.
+#[deprecated(
+    since = "0.7.2",
```

Comment:
Minor version bump if I am correct. The method has only been deprecated instead of removed, keeping the change backwards compatible.

Reply:
Ah, we've been following Rust's quasi-standard _minor changes pre-`1.0` are patches_ policy, so as a backwards compatible, non-behavior-breaking change we'd bump this as `0.7.2` instead of `0.8.0`.

Reply:
I meant patch. I’m just too tired haha :)